perfdash/perfdash.go

```diff
@@ -46,10 +47,14 @@ var (
 
 	// Storage Service Bucket and Path flags
 	logsBucket = pflag.String("logsBucket", "kubernetes-jenkins", "Name of the data bucket")
-	logsPath = pflag.String("logsPath", "logs", "Path to the logs inside the logs bucket")
+	logsPath   = pflag.String("logsPath", "logs", "Path to the logs inside the logs bucket")
 
 	// Google GCS Specific flags
 	credentialPath = pflag.String("credentialPath", "", "Path to the gcs credential json")
+
+	// AWS S3 Specific flags
+	s3Mode    = pflag.Bool("s3Mode", false, "If true, download metrics from AWS S3")
+	awsRegion = pflag.String("awsRegion", "us-west-2", "AWS region where s3 bucket lives")
```

Comment:
Can we call this `aws-region`?
```suggestion
	awsRegion = pflag.String("aws-region", "us-west-2", "AWS region of the S3 bucket")
```